src/Microsoft.AspNetCore.Mvc.Core/Infrastructure/IErrorDescriptorProvider.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+
+namespace Microsoft.AspNetCore.Mvc.Infrastructure
+{
+    public interface IErrorDescriptorProvider
+    {
+        int Order { get; }
+
+        void OnProvidersExecuting(ErrorDescriptionContext context);
+
+        void OnProvidersExecuted(ErrorDescriptionContext context);
```

Comment:
Should we simplify this down to one method? I don't see much value in spreading this project more